Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -638,14 +638,23 @@ static wxColour blend50(const wxColour& c1, const wxColour& c2)
   return a << 24 | b << 16 | g << 8 | r;
 }
 
-void CGameListCtrl::SetBackgroundColor()
+static wxColour ContrastText(const wxColour& bgc)
+{
+  // Luminance threshold to determine whether to use black text on light background
+  static constexpr int LUM_THRESHOLD = 186;
+  int lum = (((0.299 * bgc.Red()) + ((0.587 * bgc.Green()) + (0.114 * bgc.Blue()))));
```

Comment:
There are unnecessarily many parentheses here. I would prefer:

`int lum = 0.299 * bgc.Red() + 0.587 * bgc.Green() + 0.114 * bgc.Blue();`

But I suppose this also would be fine:

`int lum = (0.299 * bgc.Red()) + (0.587 * bgc.Green()) + (0.114 * bgc.Blue());`